javascript/index.js

```diff
@@ -14,52 +15,122 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  getMinutes(); //return minutes
+  getSeconds()// return seconds
+  twoDigitsNumber(minutes)
+  twoDigitsNumber(seconds)
+
+  minDec.innerHTML = minutes[0]
+  minUni.innerHTML = minutes[1]
+  secDec.innerHTML = seconds[0]
+  secUni.innerHTML = seconds[1]
+
 }
 
 function printMinutes() {
-  // ... your code goes here
+  getMinutes(); //return minutes
+  twoDigitsNumber(minutes)
+
+  minDec.innerHTML = minutes[0]
+  minUni.innerHTML = minutes[1]
 }
 
 function printSeconds() {
-  // ... your code goes here
+  getSeconds()// return seconds
+  twoDigitsNumber(seconds)
+
+  secDec.innerHTML = seconds[0]
+  secUni.innerHTML = seconds[1]
 }
 
 // ==> BONUS
 function printMilliseconds() {
   // ... your code goes here
+  // Didn't manage to do it :(
 }
 
 function printSplit() {
-  // ... your code goes here
+  
+  let splitBtn = document.getElementsByClassName("split");
+  
+  splitBtn.addEventListener("click", function(){
+    event.preventDefault();
+    let recordedTime = splitClick() // return 12:34
+    let newLi = document.createElement("li");
+    newLi.innerHTML = recordedTime;
+    splits.appendChild(newLi);
+  })
 }
 
 function clearSplits() {
-  // ... your code goes here
+  let resetBtn = document.getElementsByClassName("reset");
+
+  resetBtn.addEventListener("click", function(){
+    event.preventDefault();
+    minDec.innerHTML = "0"
+    minUni.innerHTML = "0"
+    secDec.innerHTML = "0"
+    secUni.innerHTML = "0"
+
+    while (splits.firstChild) {
+      splits.removeChild(splits.firstChild)
+    }
+  }
 }
+  
 
 function setStopBtn() {
-  // ... your code goes here
+  btnLeft.addEventListener("click", function (){
+    if (btnLeft.classList.contains("start"){
+      btnLeft.classList.remove("start");
+      btnLeft.classList.add("stop");
+      btnLeft.innerText = "STOP";  
+    }
+  }
 }
 
 function setSplitBtn() {
-  // ... your code goes here
+  btnLeft.addEventListener("click", function (){
+    if (btnLeft.classList.contains("start"){
+      btnRight.classList.remove("reset");
+      btnRight.classList.add("split");
+      btnRight.innerText = "SPLIT";  
+    }
+  }
 }
 
+
 function setStartBtn() {
-  // ... your code goes here
+  btnLeft.addEventListener("click", function (){
+    if (btnLeft.classList.contains("stop"){
+      btnLeft.classList.remove("stop");
+      btnLeft.classList.add("start");
+      btnLeft.innerText = "START";  
+    }
+  }
 }
 
+
+
 function setResetBtn() {
-  // ... your code goes here
+  btnLeft.addEventListener("click", function (){
```

Comment:
In these functions you are having some syntax errors that are breaking the code.
1. you are not closing the conditional of the if statement. Missing a ")" in Line 117.
2. you are not closing the addEventListener. Missing a ")" in Line 122.
This is happening in all functions.